srcpkgs/routinator/template

```diff
@@ -12,15 +13,30 @@ changelog="https://raw.githubusercontent.com/NLnetLabs/routinator/main/Changelog
 distfiles="https://github.com/NLnetLabs/routinator/archive/v${version}.tar.gz"
 conf_files="/etc/routinator/routinator.conf"
 checksum=00f825c53168592da0285e8dbd228018e77248d458214a2c0f86cd0ca45438f5
+system_accounts="_routinator"
+_routinator_homedir="/var/lib/routinator"
+make_dirs="/var/lib/routinator 0755 _routinator _routinator"
 
 case "$XBPS_TARGET_MACHINE" in
 	x86_64*|i686*|arm*|aarch64*) ;;
 	*) broken="ftbfs in ring" ;;
 esac
 
+post_build() {
+	pip install --user -r doc/manual/source/requirements.txt
```

Comment:
use hostmakedepends, pip is not deterministic and will pollute the masterdir

Reply:
What do I do if the packages are not in the void repos? I guess I could add them myself